File: dart/lib/model/v3_fare_estimate_result_status.dart
part of swagger.api;

class V3FareEstimateResultStatus {
  
  int statusCode = null;

  String message = null;

  V3FareEstimateResultStatus();

  @override
  String toString() {
    return 'V3FareEstimateResultStatus[statusCode=$statusCode, message=$message, ]';
  }

  V3FareEstimateResultStatus.fromJson(Map<String, dynamic> json) {
    if (json == null) return;
    statusCode = json['StatusCode'];
    message = json['Message'];
  }

  Map<String, dynamic> toJson() {
    return {
      'StatusCode': statusCode,
      'Message': message
     };
  }

  static List<V3FareEstimateResultStatus> listFromJson(List<dynamic> json) {
    return json == null ? new List<V3FareEstimateResultStatus>() : json.map((value) => new V3FareEstimateResultStatus.fromJson(value)).toList();
  }

  static Map<String, V3FareEstimateResultStatus> mapFromJson(Map<String, Map<String, dynamic>> json) {
    var map = new Map<String, V3FareEstimateResultStatus>();
    if (json != null && json.length > 0) {
      json.forEach((String key, Map<String, dynamic> value) => map[key] = new V3FareEstimateResultStatus.fromJson(value));
    }
    return map;
  }
}

File: java-vertx/src/main/java/io/swagger/server/api/model/V3FareEstimateResultStatus.java
package io.swagger.server.api.model;

import java.util.Objects;
import com.fasterxml.jackson.annotation.JsonProperty;
import com.fasterxml.jackson.annotation.JsonCreator;
import io.vertx.core.json.JsonObject;
import io.vertx.codegen.annotations.DataObject;
/**
 * V3FareEstimateResultStatus
 */

@javax.annotation.Generated(value = "io.swagger.codegen.v3.generators.java.JavaVertXServerCodegen", date = "2024-04-14T03:35:56.953023300+10:00[Australia/Sydney]")



public class V3FareEstimateResultStatus   {
  private Integer statusCode = null;

  private String message = null;

  public V3FareEstimateResultStatus statusCode(Integer statusCode) {
    this.statusCode = statusCode;
    return this;
  }

  /**
   * Get statusCode
   * @return statusCode
   **/
    public Integer getStatusCode() {
    return statusCode;
  }

  public void setStatusCode(Integer statusCode) {
    this.statusCode = statusCode;
  }

  public V3FareEstimateResultStatus message(String message) {
    this.message = message;
    return this;
  }

  /**
   * Get message
   * @return message
   **/
    public String getMessage() {
    return message;
  }

  public void setMessage(String message) {
    this.message = message;
  }


  @Override
  public boolean equals(java.lang.Object o) {
    if (this == o) {
      return true;
    }
    if (o == null || getClass() != o.getClass()) {
      return false;
    }
    V3FareEstimateResultStatus v3FareEstimateResultStatus = (V3FareEstimateResultStatus) o;
    return Objects.equals(this.statusCode, v3FareEstimateResultStatus.statusCode) &&
        Objects.equals(this.message, v3FareEstimateResultStatus.message);
  }

  @Override
  public int hashCode() {
    return Objects.hash(statusCode, message);
  }

  @Override
  public String toString() {
    StringBuilder sb = new StringBuilder();
    sb.append("class V3FareEstimateResultStatus {\n");
    
    sb.append("    statusCode: ").append(toIndentedString(statusCode)).append("\n");
    sb.append("    message: ").append(toIndentedString(message)).append("\n");
    sb.append("}");
    return sb.toString();
  }

  /**
   * Convert the given object to string with each line indented by 4 spaces
   * (except the first line).
   */
  private String toIndentedString(java.lang.Object o) {
    if (o == null) {
      return "null";
    }
    return o.toString().replace("\n", "\n    ");
  }
}
